lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -380,13 +380,14 @@ private JourneyResponse buildF2FNoMatchResponse(
         if (mitigatedCI.isPresent()) {
             var mitigationRoute = ciMitUtilityService.getMitigatedCiJourneyStep(mitigatedCI.get());
             if (mitigationRoute.isPresent()) {
-                if (mitigationRoute.get().getJourney().equals(JOURNEY_ENHANCED_VERIFICATION_PATH)) {
-                    return JOURNEY_ENHANCED_VERIFICATION_F2F_FAIL;
+                JourneyResponse journeyResponse = mitigationRoute.get();
+                if (!journeyResponse.getJourney().equals(JOURNEY_ENHANCED_VERIFICATION_PATH)) {
```

Comment:
Might be slightly safer to switch this around - if `getJourney()` was ever null we'd get a NPE. If you do it the other way around you won't. So:
`if (!JOURNEY_ENHANCED_VERIFICATION_PATH.equals(journeyResponse.getJourney()))`